src/index.html

```diff
@@ -3,9 +3,29 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
+    <title>Leaderboard</title>
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+    <main>
+        <h1>Leaderboard</h1>
```

Comment:
- [ ] As shown in the image below, your project design is missing the section highlighted in red, you are required to add this section as it is part of the wireframe design given. It is important to pay attention to details when designing as a frontend developer. 👍 ✨ 
- project wireframe
![wf](https://github.com/As1imwe-Mark/LeaderBoard/assets/89055502/aba206c6-3e4c-4810-91d9-91197b4b5a47)